### DISCUSSION TITLE
Outgoing DNS request localhost

### DISCUSSION THREAD

**Discussion by sleepingbel** [Security-Onion-Solutions/securityonion#2586] [Unsupported Versions] [answered]
Hello,

ISO install standalone 2.3.10 upgraded to 2.3.21

I have seen in my internal DNS server and my pyhole server that security onion sends daily ± 2700 localhost dns requests .
I have read the topic [https://github.com/Security-Onion-Solutions/securityonion/discussions/2493](url) 

I added the ip of my security onion and the local docker address to the hosts file.

The hosts file already contains a number of references, like hostname.localdomain and localhost.localdomain.

Is it necessary to convert the entry localdomain to your effective domain name in the existing references or do we need to ad new references other than those in topic 2493 to stop the DNS requests?

I have not found anything about this in the documentation and I have mentioned the only similar topic above.

Regards

Bart

**Comment by DebianGuru** [comment]
Bart,

Have you made any progress on this?
I'm seeing the same thing.  I'm seeing, 1,500 "ET DNS DNS Lookup for localhost.DOMAIN.TLD " alerts per hour in SOC.  TCPDump reveals there is traffic from the management port of my SO box to my DNS server looking for "localhost."  
My /etc/hosts file has a line like:

127.0.0.1   SecurityOnion2 SecurityOnion2.localdomain localhost localhost.localdomain localhost4 localhost4.localdomain

and /etc/nsswitch.conf has:

hosts:          files mdns4_minimal [NOTFOUND=return] dns
networks:       files

I believe this should be looking to the /etc/hosts file first, where localhost is defined.  Not sure why it's banging my DNS server to death. 

Any help is appreciated.

**Reply by sleepingbel** [reply]
Hello,

I have added the record "172.17.0.1	docker.localhost" to my hosts file and changed all the "localdomain" records to my domainname.

The dns record are still there.

I think that this dns record is comming from one of the docker stations. But I don't know which one.

Regards

Bart

**Reply by TOoSmOotH** [reply]
It's the so-kratos docker container. We are looking into why it looks up localhost every 15s.

**Reply by DebianGuru** [reply]
I don't know if I have something else going on.  I'm seeing about 20+ per minute.

**Reply by TOoSmOotH** [reply]
I think it depends on the amount of users connected. You can tail the kratos log and see what is going on.

**Comment by TOoSmOotH** [comment, marked as answer]
This is addressed in 2.3.30.

**Reply by DebianGuru** [reply]
Thanks for the update.  is 2.3.30 ready for release yet?  If not, do you know when?  Thanks again.
